plenum/server/node.py

```diff
@@ -2950,6 +2945,13 @@ def start_catchup(self, just_started=False):
         self.ledgerManager.catchup_ledger(self.ledgerManager.ledger_sync_order[0],
                                           request_ledger_statuses=not just_started)
 
+    def start_catchup(self, just_started=False):
```

Comment:
Can we create simple unit tests to check possibility to start catchup depending on the current state?